src/assets/TokenListController.ts

```diff
@@ -1,7 +1,7 @@
 import contractMap from '@metamask/contract-metadata';
 import type { Patch } from 'immer';
 import { Mutex } from 'async-mutex';
-import { AbortController } from 'abort-controller';
+import AbortController from 'abort-controller';
```

Comment:
Let's use a `// eslint-disable-next-line` directive to avoid modifying the ESLint config

Reply:
Ok. Can do. Just wondering though is that a lint rule we want to preserve? It feels like we may have opted into it by default?

Reply:
Yeah we probably did, but the defaults are generally reasonable, and as a matter of expediency discussions about which lint rules we should use should be had on [our ESLint config repo](https://github.com/MetaMask/eslint-config). For the purposes of this PR, I think it's best if we only concern ourselves about how we disable this particular rule for this particular instance.

Reply:
Although, now that we're here, I think that rule is good, because consider a module with both named and default exports. Many such packages exist, and the named and default exports are usually different. If you import the default export under the name of a named export, you find yourself in an annoying place if you ever also want to use the named export, since one of the names have to change. That seems bad to me!

This polyfill package is an exception in that the default and named export are the same thing, and IMO we should call them both `AbortController`.

Reply:
Makes sense. I've swapped out the config change for a `eslint-disable-next-line`